deploy/complete/helm-chart/setup/requirements.yaml

```diff
@@ -1,26 +1,31 @@
 dependencies:
   # Prometheus
   - name: prometheus
-    version: 10.3.1
+    version: 11.1.2
     condition: prometheus.enabled
     repository: https://kubernetes-charts.storage.googleapis.com
   # Grafana
   - name: grafana
-    version: 4.4.0
+    version: 5.0.13
     condition: grafana.enabled
     repository: https://kubernetes-charts.storage.googleapis.com
   # HPA Metrics
   - name: metrics-server
-    version: 2.9.0
+    version: 2.11.1
     condition: metrics-server.enabled
     repository: https://kubernetes-charts.storage.googleapis.com
   # Ingress Controller
   - name: nginx-ingress
-    version: 1.29.5
+    version: 1.36.3
     condition: nginx-ingress.enabled
     repository: https://kubernetes-charts.storage.googleapis.com
   # Service Catalog
   - name: catalog
     version: 0.3.0-beta.2
     condition: catalog.enabled
     repository: https://svc-catalog-charts.storage.googleapis.com 
+  # cert-manager
+  - name: cert-manager
+    version: 0.15.0-alpha.0
+    condition: cert-manager.enabled
+    repository: https://charts.jetstack.io
```

Comment:
This requires `helm repo add ...` in the docs for setup, right?

Reply:
No need to helm repo add. the `helm update dependency setup` get automatically with current helm client, for both, helm2 and helm3.

Reply:
Docs changed to remove the helm repo add entry.

Reply:
👍 can you remove this from the src/docs then?